tests/specs/foundry/foundry-spec.k.check.expected

```diff
@@ -66,7 +66,7 @@ module FOUNDRY-SPEC
                    ( 0 => ?_PC_CELL_acd26fc5 )
                  </pc>
                  <gas>
-                   ( infGas ( _VGAS ) => ?_GAS_CELL_acd26fc5 )
+                   ( #gas ( _VGAS ) => ?_GAS_CELL_acd26fc5 )
```

Comment:
You can see here that unparsing of the generated specification is fixed now, because `#gas` definition is included in default builds of KEVM now.